versioned_docs/version-7.x/getting-started.md

```diff
@@ -114,9 +117,25 @@ When you use a navigator (such as stack navigator), you'll need to follow the in
 
 :::
 
-## Wrapping your app in `NavigationContainer`
+## Setting up React Navigation
+
+Once you've installed and configured the dependencies, you can move on to setting up your project to use React Navigation.
+
+When using React Navigation, you configure [**navigators**](glossary-of-terms.md#navigator) in your app. Navigators handle the transition between screens in your app and provide UI such as header, tab bar etc.
+
+There are 2 primary ways to configure the navigators:
+
+### Static configuration
 
-Now, we need to wrap the whole app in `NavigationContainer`. Usually you'd do this in your entry file, such as `index.js` or `App.js`:
+The static configuration API has reduced boilerplate and simplifies things such as TypeScript types and deep linking. If you're starting a new project or are new to React Navigation, this is the **recommended way** to set up your app.
```

Comment:
```suggestion
The static configuration API has reduced boilerplate and simplifies things such as TypeScript types and deep linking. It might be considered a good way to quickly start the development of a new app, not putting too much energy into configuring the navigation structure. Although it may not provide a sufficient level of flexibility, that should be more than enough for the majority of the applications. 
```